posthog/year_in_posthog/2022.html

```diff
@@ -0,0 +1,307 @@
+        </svg>
+    </a>
+    <a target="_blank" href="https://www.linkedin.com/sharing/share-offsite/?url={{ page_url | urlencode }}">
+        <svg style="width:24px;height:24px" viewBox="0 0 24 24">
+            <path fill="white"
+                  d="M19 3A2 2 0 0 1 21 5V19A2 2 0 0 1 19 21H5A2 2 0 0 1 3 19V5A2 2 0 0 1 5 3H19M18.5 18.5V13.2A3.26 3.26 0 0 0 15.24 9.94C14.39 9.94 13.4 10.46 12.92 11.24V10.13H10.13V18.5H12.92V13.57C12.92 12.8 13.54 12.17 14.31 12.17A1.4 1.4 0 0 1 15.71 13.57V18.5H18.5M6.88 8.56A1.68 1.68 0 0 0 8.56 6.88C8.56 5.95 7.81 5.19 6.88 5.19A1.69 1.69 0 0 0 5.19 6.88C5.19 7.81 5.95 8.56 6.88 8.56M8.27 18.5V10.13H5.5V18.5H8.27Z"/>
+        </svg>
+    </a>
+    <a target="_blank"
+       href="https://twitter.com/intent/tweet?text={{ 'No hedgehogs were harmed in the course of this year on @posthog' }}&url={{ page_url | urlencode }}&hashtags=YearInPostHog,NoHibernation,HedgeHustle,AchievementUnlocked">
+        <svg style="width:24px;height:24px" viewBox="0 0 24 24">
+            <path fill="white"
+                  d="M22.46,6C21.69,6.35 20.86,6.58 20,6.69C20.88,6.16 21.56,5.32 21.88,4.31C21.05,4.81 20.13,5.16 19.16,5.36C18.37,4.5 17.26,4 16,4C13.65,4 11.73,5.92 11.73,8.29C11.73,8.63 11.77,8.96 11.84,9.27C8.28,9.09 5.11,7.38 3,4.79C2.63,5.42 2.42,6.16 2.42,6.94C2.42,8.43 3.17,9.75 4.33,10.5C3.62,10.5 2.96,10.3 2.38,10C2.38,10 2.38,10 2.38,10.03C2.38,12.11 3.86,13.85 5.82,14.24C5.46,14.34 5.08,14.39 4.69,14.39C4.42,14.39 4.15,14.36 3.89,14.31C4.43,16 6,17.26 7.89,17.29C6.43,18.45 4.58,19.13 2.56,19.13C2.22,19.13 1.88,19.11 1.54,19.07C3.44,20.29 5.7,21 8.12,21C16,21 20.33,14.46 20.33,8.79C20.33,8.6 20.33,8.42 20.32,8.23C21.16,7.63 21.88,6.87 22.46,6Z"/>
+        </svg>
+    </a>
+</div>
+<main class="px-4 pb-4" role="main">
+    <div class="left column">
+        <div>This year, I'm {% if badge == "astronaut" %}an{% else %}a{% endif %}</div>
+        <img width="326" height="326" class="badge my-8" src="{% static image %}"
+             alt="image for this badge {{ badge }}. An illustration of a hedgehog who is a {{ human_badge }}"/>
+        <h1 class="highlight text-2xl">{{ human_badge }}</h1>
+    </div>
+    <div class="right column">
+        <div class="muted">You earned this badge because...</div>
+        <div class="highlight text-xl my-8">{{ explanation }}</div>
+        <div class="stats">
+            {% for stat in stats %}
+                <div class="stat px-1">
+                    <div class="highlight text-2xl">{{ stat.count }}</div>
+                    <div>{{ stat.description }}</div>
+                </div>
+            {% endfor %}
+        </div>
+    </div>
+</main>
+<footer class="footer" role="contentinfo">
+    <div class="footer-content pl-8 text-xl">
+
+        <span>My year in&nbsp;&nbsp;</span>
+        <img class="logo" src="{% static "logomark.png" %}"
+             alt="PostHog logo"/>
+        <span>&nbsp;PostHog</span>
```

Comment:
This uses the old (unrounded) logomark + the font is wrong (the logotype is Matter SQ).  We should be using the logo as one package.